test/entry.ts

```diff
@@ -56,6 +56,14 @@ function withinExpectedTimeBoundaries(result?: Date): boolean {
   return false;
 }
 
+function nanosecondsToDate(timestamp: entryTypes.Timestamp) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const seconds = (timestamp as any).seconds;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const nanos = (timestamp as any).seconds;
```

Comment:
is this supposed to be `.nanos` instead of `.seconds`?

Reply:
Ah, right! Good catch!